viadot/sources/sap_rfc.py

```diff
@@ -257,7 +258,14 @@ def __init__(
 
         self._con = None
         DEFAULT_CREDENTIALS = local_config.get("SAP").get("DEV")
-        credentials = kwargs.pop("credentials", None) or DEFAULT_CREDENTIALS
+
+        credentials = kwargs.pop("credentials", None)
+        if credentials is None:
+            credentials = DEFAULT_CREDENTIALS
+            logger.warning(
+                "Your credentials will use DEV environment. If you would like to use different one - please specified it in 'sap_credentials' variable inside the flow."
+            )
+
         if credentials is None:
             raise CredentialError("Missing credentials.")
```

Comment:
It is the same if statement - maybe add this under the first one.

Reply:
Done.